i18n/locales/en.ts

```diff
@@ -32,11 +32,9 @@ const translations = {
     project: {
       title: 'Built with fun',
       paragraph_one:
-        'Saving Satoshi is a light-hearted first point of contact for developers who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image (aka AI) tools like Midjourney.',
+        'Saving Satoshi is a light-hearted first point of contact for coders who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image tools like Midjourney.',
       paragraph_two:
-        'This version of the project is currently in a demo state with a goal of getting feedback on the overall experience and providing a proof of concept for early users. In light of this, user progress is stored locally in the browser and the story along with images are not finalized.',
-      paragraph_three:
-        'Based on the feedback we receive, we will then work towards an official release (we call it V1) that includes 2-3 full lesson chapters, accounts, and more. 10 chapters are planned out and will be released over time as they are completed. Additional features may include collaborative learning, badges, extra-challenging side quests, and more.',
+        'This free and open source software (FOSS) project is in continuous development. Our goal is to take an iterative approach to incorporate feedback as we release new chapters. There are currently 10 chapters mapped out, however more may be added as the project continues to grow.',
```

Comment:
nit:
```suggestion
        'This free and open source software (FOSS) project is in steady development. We aim to take an iterative approach to incorporate feedback as we release new chapters. There are currently ten chapters mapped out. However, more may be added as the project continues to grow.',
```

Reply:
Thanks for zeroing on this section. I incorporated some of your suggestions but also updated the copy over all.